torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8657,6 +8658,9 @@ def ref_pairwise_distance(input1, input2):
                # INTERNAL ASSERT FAILED at "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":252,
                # please report a bug to PyTorch.
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit', dtypes=[torch.float32]),
+               # broadcast_arrays returns an iterable of tensors.
+               # AttributeError: 'tuple' object has no attribute 'dtype'
+               DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_jit_alias_remapping'),
```

Comment:
How did this not fail previously?

Reply:
I believe that this test only runs if the opinfo has an alias entry, that's why it didn't run/fail before